app/assets/stylesheets/patternfly_overrides.scss

```diff
@@ -741,3 +741,8 @@ table.c3-tooltip td {
     }
   }
 }
+
+/* overrides col-md-2 from patternfly (width: 16.6666666667%) */
```

Comment:
@DavidResende0 do we need this change?

Reply:
it prevents the title from wrapping onto a new line which was part of the above issue but I don't really see the text wrapping as a big deal myself

Reply:
the reason I made the change for the whole `col-md-2` class was because I thought it best that everything line up on the page

Reply:
Instead of changing for all places, try to add a new class name for that field and add sass css, something like this
```
     #report_list_div {
       .Newclass {
         ## Add css here
        }
      }